sqlglot/dialects/dialect.py

```diff
@@ -185,6 +182,10 @@ class Dialect(metaclass=_Dialect):
     # True means a / b is integer division if both a and b are integers.
     TYPED_DIVISION = False
 
+    # False means 1 / 0 throws an error.
+    # True means 1 / 0 returns null.
```

Comment:
what about where it returns nan?

Reply:
is there a difference between 1 / 0 and 1.0 / 0?

Reply:
1 / 0:

- ClickHouse: ?
- DuckDB: nan
- Hive: null
- MySQL: null
- Spark: ?
- Spark2: ?
- SQLite: null

@GeorgeSittas - can you fill in the blanks for clickhouse and spark?

Reply:
DuckDB, ClickHouse and Spark all return `NULL` for both `1 / 0` and `1.0 / 0`

Reply:
You sure about that for duckdb? I might be on an old version

Reply:
Ah, I'm calling the `.df()` method, that's why

Reply:
ok, looks like nothing return nan.